templates/default/onlinefs-site.xml.erb

```diff
@@ -26,6 +28,8 @@
     <topicPattern><%= node['onlinefs']['kafka_consumer']['topic_pattern'] %></topicPattern>
     <topicList><%= node['onlinefs']['kafka_consumer']['topic_list'] %></topicList>
     <pollTimeoutMs><%= node['onlinefs']['kafka_consumer']['poll_timeout_ms'] %></pollTimeoutMs>
+    <ronDbGroupId><%= node['onlinefs']['kafka_consumer']['ron_db_group_id'] %></ronDbGroupId>
+    <vectorDbGroupId><%= node['onlinefs']['kafka_consumer']['vector_db_group_id'] %></vectorDbGroupId>
```

Comment:
No you can't do this, because for bring your own Kafka setup, users provide their own `onlinefs-kafka`. That file might need to include the `group.id`, especially in multi-region setups (the group-id is going to be different depending on the region).

I think we should have multiple `onlinefs-kafka.properties` file, one for the consumer groups of rondb and one for the consumer group of elastic.